framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -370,6 +370,11 @@ private boolean isContainedWildcard(
             AnnotatedTypeMirror outsideLowerBound,
             boolean canBeCovariant) {
 
+        if (inside.getKind() == TypeKind.WILDCARD
```

Comment:
Why is this only for wildcards? Isn't it enough if `inside.equals(outside)` returns true?